### PR TITLE
fix(http-validation): clarify the location of the validation error

### PR DESCRIPTION
**Pull request by MickVanDuijn**

### CODE REVIEW

**Comment by MickVanDuijn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #27** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/27" target="_blank">Graphite</a>  👈
    * **PR #28** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/28" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/27?utm_source=stack-comment).

**Review by pevisscher — APPROVED** [review]

**Comment by MickVanDuijn** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/27?utm_source=stack-comment).


Job Reference: E627JC01mZxZaCEjTdUI

**Pull request merged**
